omero/developers/Web/Webclient.txt

```diff
@@ -0,0 +1,90 @@
+OMERO.webclient
+===============
+
+The webclient Django app provides the main OMERO.web client UI.
+
+Data from OMERO is rendered to html using Django templates before being
```

Comment:
maybe it should say: data retrieved from OMERO server using python gateway (``:pythondoc:`omero.gateway <omero/omero.gateway.html>``) is rendered using Django templates as an HTML page.